src/polyswarm_api/__init__.py

```diff
@@ -99,11 +41,13 @@ def __init__(self, key, uri="https://api.polyswarm.network/v1", get_limit=100,
 
         self.network = "prod"
         self.portal_uri = "https://polyswarm.network/scan/results/"
-
+        self.api_uri = "https://api.k.polyswarm.network"
```

Comment:
What is this `.k` domain vs api.polyswarm.network? API URI? This should just be `self.uri` unless I'm missing something entirely.

Reply:
ok, this is a little confusing because the default value includes `v1` but if you use the `-u` flag from command line someone could use consumer url or a url without v1.